docs/src/communication.md

```diff
@@ -78,7 +78,7 @@ pass more or fewer arguments, pass your arguments as an array:
 handle(w, "event") do count, values, message
   # ...
 end
-@js w Blink.msg("event", [1, ['a','b'], "Hi"]);
+@js w Blink.msg("press", [1, ['a','b'], "Hi"]);
```

Comment:
Oops, I actually think this was the wrong fix. The problem is actually with the way the arguments to the `"event"` handler are declared.

I think what this example was _trying_ to show is that you can to use argument destructuring in order to pass multiple arguments to a handler..
I just opened https://github.com/JuliaGizmos/Blink.jl/pull/238 to fix it. :)  Thanks for responding to the issue in the first place! :)

Reply:
Ah, I didn't even notice that.

🤷‍♀️